data/biobank-directory/molgenis.csv

```diff
@@ -115,6 +115,7 @@ Collections,,access_joint_project,Access via Join Projects to,ontology_array,,fa
 Collections,,access_description,Access Description,text,,false,,Short description of the access rules.,,,,,,,,,
 Collections,,access_uri,Access URI,hyperlink,,false,,URI describing the access policy.,,,,,,,,,
 Collections,,sop,PD/SOPs,ontology_array,,false,,Availability of Process Descriptions (PDs) and/or Standard Operating Procedures (SOPs).,DirectoryOntologies,SOPs,,${label},,,,,
+Collections,,study,Study,ref,,false,,A link to a Study during which the collection was generated,,Studies,,${title},,,,,

```

Comment:
Sorry, forgot to mention it earlier, but now the study is shown under the Policies header. I think it's better to move this to the "Belongs to' one. So after also_known.

Reply:
moved